string controller_name  # Name of the controller to switch to
---
bool success
string message 